pkg/lightning/backend/tidb/tidb.go

```diff
@@ -309,6 +334,11 @@ type tidbBackend struct {
 	// affecting the cluster too much.
 	maxChunkSize uint64
 	maxChunkRows int
+	// implement stmtCache to improve performance, especially when the downstream is TiDB
+	stmtCache      *kvcache.SimpleLRUCache
```

Comment:
as long as the table is importing, the prepared stmt is needed, no need such LRU

Reply:
The reason to use LRU here is that the batch insert prepared statement may not be the same. where the insert/replace statement is constrained by `logical-import-batch-size` , the actual SQL executed might be longer or shorter, depending on the actual content imported.

Reply:
depends on row size is unstable for prepared stmt, in worse case, every prepared stmt is different. we can ignore batch-size in this case, and only honors batch-rows, cc @lance6716

Reply:
the default `logical-import-batch-rows` is 65536, usually this batch size is too large, should we lower the default value of `logical-import-batch-rows` to some reasonable number like 128/256/512?

Reply:
Maybe we can ask the user that request this feature what's the root cause https://github.com/pingcap/tidb/issues/46607 . If the prepared statement can solve that problem, we can ignore these configurations. @D3Hunter and I are busy this week, please help to contact the issue author.

Reply:
Not the author of #46607, but we've got customer which database is unable to handle 96 KB transactions thus requiring the limit to be lowered.

The `-batch-size` does not need to be very precise. It is mainly used to ensure (number of rows) &times; (size of row) to be maintained at a constant limit. You should *not* ignore the batch-size entirely because the same limit is shared among all tables and the constraint should be reasonable for both wide and narrow tables.